Unable to build according to the documented guides

Following the instructions in https://github.com/vmware-tanzu/octant/blob/master/HACKING.md yields an error from the go run build.go ci-quick command:
```
npm ERR! errno 130
npm ERR! @angular/cli@8.3.4 postinstall: `node ./bin/postinstall/script.js`
npm ERR! Exit status 130
npm ERR!
npm ERR! Failed at the @angular/cli@8.3.4 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-17T19_52_39_751Z-debug.log
2020/01/17 19:52:39 web-deps: exit status 130
exit status 1
```
A quick look at the logs reveals this:
```
8560 verbose linkBins [ { ng: './bin/ng' },
8560 verbose linkBins   '/root/octant/web/node_modules/.bin',
8560 verbose linkBins   false ]
8561 info lifecycle @angular/cli@8.3.4~install: @angular/cli@8.3.4
8562 info lifecycle @angular/cli@8.3.4~postinstall: @angular/cli@8.3.4
8563 verbose lifecycle @angular/cli@8.3.4~postinstall: unsafe-perm in lifecycle false
8564 verbose lifecycle @angular/cli@8.3.4~postinstall: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/root/octant/web/node_modules/@angular/cli/node_
modules/.bin:/root/octant/web/node_modules/.bin:/root/go/bin:/usr/lib/go-1.13/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:
:/bin
8565 verbose lifecycle @angular/cli@8.3.4~postinstall: CWD: /root/octant/web/node_modules/@angular/cli
8566 silly lifecycle @angular/cli@8.3.4~postinstall: Args: [ '-c', 'node ./bin/postinstall/script.js' ]
8567 silly lifecycle @angular/cli@8.3.4~postinstall: Returned: code: 130  signal: null
8568 info lifecycle @angular/cli@8.3.4~postinstall: Failed to exec postinstall script
8569 verbose teardown shutting down workers.
8570 info teardown Done in 0.001s
8571 verbose stack Error: @angular/cli@8.3.4 postinstall: `node ./bin/postinstall/script.js`
8571 verbose stack Exit status 130
8571 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
8571 verbose stack     at EventEmitter.emit (events.js:189:13)
8571 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
8571 verbose stack     at ChildProcess.emit (events.js:189:13)
8571 verbose stack     at maybeClose (internal/child_process.js:970:16)
8571 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
8572 verbose pkgid @angular/cli@8.3.4
8573 verbose cwd /root/octant/web
8574 verbose Linux 4.15.0-74-generic
8575 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "ci"
8576 verbose node v10.15.3
8577 verbose npm  v6.4.1
8578 error code ELIFECYCLE
8579 error errno 130
8580 error @angular/cli@8.3.4 postinstall: `node ./bin/postinstall/script.js`
8580 error Exit status 130
8581 error Failed at the @angular/cli@8.3.4 postinstall script.
8581 error This is probably not a problem with npm. There is likely additional logging output above.
8582 verbose exit [ 130, true ]
```
This is on ubunut 18.04 running node 10.15.3 and npm 6.4.1

I was able to work around this by following these steps suggested by @GarySmith :
cd web
npm install
npm run build
cd .. 
go generate ./web
go run build.go build

Also the https://github.com/vmware-tanzu/octant/blob/master/web/README.md has build instructions that include using a make from the top level dir, I don't believe a makefile exists at this level.

I'm also seeing an error similar when setting up a new development environment:
```
? Would you like to share anonymous usage data with the Angular Team at Google u
nder
Google’s Privacy Policy at https://policies.google.com/privacy? For more details
 and
how to change this setting, see http://angular.io/analytics. (y/N) npm ERR! code ELIFECYCLE
npm ERR! errno 130
npm ERR! @angular/cli@9.0.1 postinstall: `node ./bin/postinstall/script.js`
npm ERR! Exit status 130
npm ERR! 
npm ERR! Failed at the @angular/cli@9.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jordan/.npm/_logs/2020-03-24T19_00_24_973Z-debug.log
2020/03/24 12:00:25 web-deps: exit status 130
exit status 1
```

Log info:
```
12661 verbose lifecycle @angular/cli@9.0.1~postinstall: PATH: /home/jordan/.nvm/versions/node/v13.11.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/jordan/go/src/github.com/vmware-tanzu/octant/web/node_modules/@angular/cli/node_modules/.bin:/home/jordan/go/src/github.com/vmware-tanzu/octant/web/node_modules/.bin:/home/jordan/.nvm/versions/node/v13.11.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/local/go/bin
12662 verbose lifecycle @angular/cli@9.0.1~postinstall: CWD: /home/jordan/go/src/github.com/vmware-tanzu/octant/web/node_modules/@angular/cli
12663 silly lifecycle @angular/cli@9.0.1~postinstall: Args: [ '-c', 'node ./bin/postinstall/script.js' ]
12664 silly lifecycle @angular/cli@9.0.1~postinstall: Returned: code: 130  signal: null
12665 info lifecycle @angular/cli@9.0.1~postinstall: Failed to exec postinstall script
12666 verbose teardown shutting down workers.
12667 info teardown Done in 0s
12667 info teardown Done in 0s
12668 verbose stack Error: @angular/cli@9.0.1 postinstall: `node ./bin/postinstall/script.js`
12668 verbose stack Exit status 130
12668 verbose stack     at EventEmitter.<anonymous> (/home/jordan/.nvm/versions/node/v13.11.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
12668 verbose stack     at EventEmitter.emit (events.js:315:20)
12668 verbose stack     at ChildProcess.<anonymous> (/home/jordan/.nvm/versions/node/v13.11.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
12668 verbose stack     at ChildProcess.emit (events.js:315:20)
12668 verbose stack     at maybeClose (internal/child_process.js:1026:16)
12668 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
12669 verbose pkgid @angular/cli@9.0.1
12670 verbose cwd /home/jordan/go/src/github.com/vmware-tanzu/octant/web
12671 verbose Linux 5.3.0-7642-generic
12672 verbose argv "/home/jordan/.nvm/versions/node/v13.11.0/bin/node" "/home/jordan/.nvm/versions/node/v13.11.0/bin/npm" "ci"
12673 verbose node v13.11.0
12674 verbose npm  v6.13.7
12675 error code ELIFECYCLE
12676 error errno 130
12677 error @angular/cli@9.0.1 postinstall: `node ./bin/postinstall/script.js`
12677 error Exit status 130
12678 error Failed at the @angular/cli@9.0.1 postinstall script.
12678 error This is probably not a problem with npm. There is likely additional logging output above.
12679 verbose exit [ 130, true ]
```

--- a/HACKING.md
+++ b/HACKING.md
@@ -13,9 +13,11 @@
 
     git clone git@github.com:vmware-tanzu/octant.git
     cd octant
-    go run build.go go-install  # install Go dependencies.
-    go run build.go ci-quick    # build UI, generate UI files, and create octant binary.
-    ./build/octant   # run the Octant binary you just built
+    go run build.go go-install      # install Go dependencies.
+    export NG_CLI_ANALYTICS=false   # if you want to disable Angular CLI analytics or
+    export NG_CLI_ANALYTICS=ci      # if you want to enable Angular CLI analytics
+    go run build.go ci-quick        # build UI, generate UI files, and create octant binary.
+    ./build/octant                  # run the Octant binary you just built
 
 ## Testing
 

--- a/changelogs/unreleased/792-dotNomad
+++ b/changelogs/unreleased/792-dotNomad
@@ -1,0 +1,1 @@
+Add NG_CLI_ANALYTICS instructions to HACKING

